client/test/commands/testSmartContractCommands.test.ts

```diff
@@ -465,17 +472,24 @@ describe('testSmartContractCommand', () => {
                 certificatePath: 'fakeCertificatePath',
                 privateKeyPath: 'fakePrivateKeyPath'
             };
-            fabricRuntimeConnectionMock = sinon.createStubInstance(FabricRuntimeConnection);
-            fabricRuntimeConnectionMock.connect.resolves();
-            fabricRuntimeConnectionMock.instantiateChaincode.resolves();
-            fabricRuntimeConnectionMock.getMetadata.resolves(JSON.parse('{"":{"functions":["instantiate","doubledecking","transaction4"]},"org.hyperledger.fabric":{"functions":["getMetaData"]}}'));
-            fabricRuntimeConnectionMock.getConnectionDetails.resolves(fakeRuntimeConnectionDetails);
+
+            fabricClientConnectionMock = sinon.createStubInstance(FabricClientConnection);
+            fabricClientConnectionMock.connect.resolves();
+            fabricClientConnectionMock.instantiateChaincode.resolves();
+            fabricClientConnectionMock.getMetadata.resolves(JSON.parse('{"":{"functions":["instantiate","doubledecking","transaction4"]},"org.hyperledger.fabric":{"functions":["getMetaData"]}}'));
+            fabricClientConnectionMock.getConnectionDetails.resolves(fakeConnectionDetails);
             fabricConnectionManager = FabricConnectionManager.instance();
-            getConnectionStub = mySandBox.stub(fabricConnectionManager, 'getConnection').returns(fabricRuntimeConnectionMock);
-            fabricRuntimeConnectionMock.getAllPeerNames.returns(['peerThree']);
-            fabricRuntimeConnectionMock.getAllChannelsForPeer.withArgs('peerThree').resolves(['myChannelTunnel']);
-            fabricRuntimeConnectionMock.getInstantiatedChaincode.resolves([
-                {
+            getConnectionStub = mySandBox.stub(fabricConnectionManager, 'getConnection').returns(fabricClientConnectionMock);
+
+            registryEntry = new FabricConnectionRegistryEntry();
+            registryEntry.name = 'myConnection';
+            registryEntry.connectionProfilePath = 'myPath';
+            registryEntry.managedRuntime = true;
+            getRegistryStub = mySandBox.stub(fabricConnectionManager, 'getConnectionRegistryEntry').returns(registryEntry);
+            fabricClientConnectionMock.getAllPeerNames.returns(['peerOne']);
+            fabricClientConnectionMock.getAllChannelsForPeer.withArgs('peerOne').resolves(['myEnglishChannel']);
+            fabricClientConnectionMock.getInstantiatedChaincode.resolves([
```

Comment:
i think all these fabricClientConnectionMock should be "fabricRuntimeConnectionMock" ?